application-templates/javascript/event/src/routes/event.route.js

```diff
@@ -1,13 +1,19 @@
 import { Router } from 'express';
 
 import { logger } from '../utils/logger.utils.js';
+import { post } from '../controllers/event.controller.js';
 
 const eventRouter = Router();
 
-eventRouter.post('/', async (req, res) => {
+eventRouter.post('/', async (req, res, next) => {
   logger.info('Event message received');
-  res.status(200);
-  res.send();
+  
+  try {
+    await post(req.body);
+    res.status(200).send();
```

Comment:
If we are "proxying" the request and response here (not know the real reason) to the `event.controller.js`, then it is their responsibility to define the HTTP status and response body.